src/index.html

```diff
@@ -13,6 +13,38 @@
     />
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day-sun calendar--month-length-31">
```

Comment:
The class 'calendar--month-length-31' suggests that you are trying to control the number of days in the month via a class. This is not a good practice as it mixes content with presentation. The number of days should be controlled by the back-end logic or JavaScript.